interfaces/builtin/unity7.go

```diff
@@ -194,6 +194,11 @@ dbus send
 owner @{HOME}/.local/share/mime/**   r,
 owner @{HOME}/.config/user-dirs.dirs r,
 
+# gtk settings
+owner @{HOME}/.config/gtk-2.0/gtkfilechooser.ini r,
+owner @{HOME}/.config/gtk-3.0/bookmarks r,
```

Comment:
Done and pushed (I'm not part of the team, so I'll need someone from the team to merge it). Thanks!